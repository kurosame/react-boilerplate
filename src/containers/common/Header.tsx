import React from 'react'
import styled from 'styled-components'

const Header = (): JSX.Element => <Div>Header</Div>

const Div = styled.div`
  color: white;
  background-color: red;
`

export default Header
