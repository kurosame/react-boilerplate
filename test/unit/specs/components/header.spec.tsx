import Header from '@/components/globals/Header'
import { mount } from 'enzyme'
import React from 'react'

const wrapper = mount(<Header />)

test('Match the snapshot', () => {
  expect(wrapper.html()).toMatchSnapshot()
})
