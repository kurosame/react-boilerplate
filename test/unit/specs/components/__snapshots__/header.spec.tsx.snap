// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm kEdiiQ"
  >
    Header
  </div>
</DocumentFragment>
`;
